src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,57 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int DATA_CORRECT_LENGTH = 6;
+    private static final int MINIMUM_AGE = 18;
+    private static final int MAXIMUM_AGE = 100;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        throwNullException(user, "User");
+        checkLogin(user.getLogin());
+        checkPassword(user.getPassword());
+        checkAge(user.getAge());
```

Comment:
it is better to pass `user` instance to each method so that it is responsibility of a method to check what's necessary

caller should not bother about it
```suggestion
        checkLogin(user);
        checkPassword(user);
        checkAge(user);
```